src/helpers/compose.ts

```diff
@@ -0,0 +1,22 @@
+type Func<I, O> = (input: I) => O;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Compose = (...functions: Func<any, any>[]) => Func<any, any>;
```

Comment:
Prevent writing `any` type because the types will be missed explicitly after composing.

Reply:
i used `any` type to make the type simpler, in typescript we can't write a good type for compose function, for example... take a look at the [@types/ramda ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/22a636dad575178e01f7687ad712763538452c31/types/ramda/index.d.ts#L297-L325), that's scary!